misc/make-deps.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 # setup a simple go environment
+if [[ -z "${GOBIN}" ]]; then
```

Comment:
This stuff should be in misc/make-path.sh ... can you see if that needs improving?

Reply:
Sorry, I didn't notice. I removed those lines now.